src/MdocDataTransfer.ts

```diff
@@ -0,0 +1,11 @@
+import { MdocNativeModule, mDocNativeModule } from './MdocDataTransferModule'
+
+export class MdocDataTransfer {
+  public constructor(x509Certificates: Array<string>, documentResolver: () => Record<string, unknown>) {
+    mDocNativeModule.initialize(x509Certificates, documentResolver)
+  }
```

Comment:
initialize is global right? If so i don't think this should be a class constructor, as that gives the impression that each class has separate context. I think it should be a static method, or call-once separate function

Reply:
Yes, good point. Let me rework it.